iothub/device/samples/PnpDeviceSamples/PnpHelper/PnpHelper.cs

```diff
@@ -150,6 +150,16 @@ public static string CreatePnpCommandName(string commandName, string componentNa
             return string.IsNullOrWhiteSpace(componentName) ? commandName : $"{componentName}*{commandName}";
         }
 
+        /// <summary>
+        /// Create the DPS payload to provision a device as plug and play.
+        /// </summary>
+        /// <param name="modelId">The model Id corresponding to the device.</param>
```

Comment:
The Id of the model the device adheres to for properties, telemetry, and commands.